core/src/main/java/org/axonframework/config/EventHandlingConfiguration.java

```diff
@@ -533,8 +617,9 @@ public EventHandlingConfiguration registerTokenStore(String name, Function<Confi
      * @return an Optional referencing the processor, if present.
      */
     public <T extends EventProcessor> Optional<T> getProcessor(String name) {
```

Comment:
Shouldn't all public calls be deprecated for this class?

Reply:
I think you're right. It didn't hurt that much to leave it here, but when I gave it a second thought, we can deprecate it.